server/graphql/v2/collection/ExpenseCollection.ts

```diff
@@ -12,5 +14,24 @@ export const GraphQLExpenseCollection = new GraphQLObjectType({
     nodes: {
       type: new GraphQLList(GraphQLExpense),
     },
+    aggregation: {
+      type: new GraphQLObjectType({
+        name: 'ExpenseCollectionAggregation',
+        fields: {
+          totalAmount: {
+            args: {
+              currency: {
+                type: GraphQLCurrency,
+                defaultValue: 'USD',
+              },
+            },
+            type: GraphQLAmount,
+          },
+          currencyAmounts: {
+            type: new GraphQLList(GraphQLAmount),
+          },
+        },
+      }),
+    },
```

Comment:
I would propose to send back a `totalAmount: GraphQLAmount`

And extend the `GraphQLAmount` type to support a list of amounts it's aggregating

Reply:
Or:

```
const GraphQLTotalAmount = new GraphQLObjectType({
        name: 'TotalAmount',
        fields: {
          amount: {
            type: GraphQLAmount,
         },
        amountsByCurrency: {
            type: new GraphQLList(GraphQLAmount),
          },
       }
```

```
totalAmount: new GraphQLTotalAmount()
```